### ISSUE TITLE
README is copied from ansible-s3?

### ISSUE THREAD

**Issue by thatarchguy** [crushlovely/ansible-elb#1]
It seems you copied this README from your https://github.com/crushlovely/ansible-s3 project and forgot to edit it :smile:

**Comment by pablitoc** [comment]
that is correct. Sorry for the super delayed response but my attention was taken elsewhere.  This was left as is  because I was still working out the kinks.  I will continue development on this again.
